RT_SYCL/camera.hpp

```diff
@@ -86,6 +86,12 @@ class camera {
     lens_radius = aperture/2;
   }
 
+  /** get_ray(s,t) 
+     
+      Computes ray from camera passing through 
```

Comment:
Do not repeat the source code
```suggestion
  /** Computes ray from camera passing through 
```